resources/articles/platform_adhesion/raft_speed.md

```diff
@@ -1,7 +1,7 @@
 Raft Print Speed
 ====
-This setting determines the overall speed at which the raft is printed. The actual speed to print the raft at may still be different if the speed is specialised for the base, middle or top layers.
+This setting determines the overall speed at which the raft is printed. The actual speed at which the raft is printed may differ from this if specific speeds are set for the base, middle or top layers. 
```

Comment:
Can we keep the word "still" in, to indicate the contrast like the original sentence did?